src/App.js

```diff
@@ -1,25 +1,34 @@
-import logo from './logo.svg';
-import './App.css';
+import background from '../src/assets/viary-bg.jpg'
+import Header from '../src/components/Header'
+import MemoriesPage from './pages/MemoriesPage'
+import image from './assets/images/memory_1.jpg'
+import image2 from './assets/images/memory_2.jpg'
+
+export default function App() {
+  const listOfMemories = [
+    { image, subtitle: 'Memory 1' },
+    { image: image2, subtitle: 'Memory 2' },
+  ]
 
-function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div
+      className="App"
+      style={{
+        backgroundImage: `url(${background})`,
+        backgroundPosition: 'center',
+        backgroundSize: 'cover',
+        backgroundRepeat: 'no-repeat',
+        height: '645px',
```

Comment:
A fixed height is not a good idea in most of the cases. This will make it difficult to support multiple devices/viewports.